From 6311c6b4d7ebe989447151a3e0f8b8ae0b5997d3 Mon Sep 17 00:00:00 2001
From: Sergey Poznyakoff <gray@gnu.org>
Date: Fri, 3 Aug 2018 11:47:01 +0300
Subject: [PATCH 1/1] Bugfix

* src/gdbmopen.c (gdbm_avail_table_valid_p): Fix improper use of
the GDBM_DEBUG macro.
---
 src/gdbmopen.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gdbmopen.c b/src/gdbmopen.c
index dc5a89e..908887c 100644
--- a/src/gdbmopen.c
+++ b/src/gdbmopen.c
@@ -99,7 +99,7 @@ gdbm_avail_table_valid_p (GDBM_FILE dbf, avail_elem *av, int count)
 
   if (needs_sorting && dbf->read_write)
     {
-      GDBM_DEBUG (GDBM_DEBUG_ERR, "restoring sort order");
+      GDBM_DEBUG (GDBM_DEBUG_ERR, "%s", "restoring sort order");
       qsort (av, count, sizeof av[0], avail_comp);
     }
   
-- 
2.8.3

